easybuild/easyconfigs/c/CubeGUI/CubeGUI-4.8-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,64 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2019 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Score_minus_P'
+
+name = 'CubeGUI'
+version = '4.8'
+
+homepage = 'https://www.scalasca.org/software/cube-4.x/download.html'
+description = """
+ Cube, which is used as performance report explorer for Scalasca and Score-P,
+ is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and
+ (iii) system resource. Each dimension can be represented as a tree, where
+ non-leaf nodes of the tree can be collapsed or expanded to achieve the
+ desired level of granularity.
+
+ This module provides the Cube graphical report explorer.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://apps.fz-juelich.de/scalasca/releases/cube/%(version)s/dist']
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('CubeGUI-4.8_missing_compile_file.patch', 1),
+]
```

Comment:
Could you please provide us (the Cube development team) with an example where this patch is actually needed?  AFAIK, the `compile` file is only used by the `AC_PROG_(CC|CXX)_C_O` macros and should then be copied into the tarball automatically by Autoconf.  Just adding the file after the fact will be a no-op.

Reply:
after removing `Autotools` and `preconfigopts` from the easyconfig, the need for the `compile` file is gone. I think they triggered the search for the `compile` file.

Reply:
I've removed the patch